### ISSUE TITLE
Chips uses wrong placeholder 

### ISSUE THREAD

**Issue by superamadeus** [Dogfalo/materialize#4248]
### Description
When initializing a chips component with 'placeholder' and 'secondaryPlaceholder', the wrong placeholder is used.

### Reproduction Steps
Initialize the chips component with 'placeholder' and 'secondaryPlaceholder'.

```typescript
    $( document ).ready(function() {
      $('.chips').material_chip({
        placeholder: 'Primary.',
        secondaryPlaceholder: 'Secondary',
      });
    });
```

In the resulting chips input, the secondary placeholder will be used when there are no tags, and the primary placeholder will be used when there are tags, which is the opposite of what the documentation states.

### Screenshots / Codepen
http://codepen.io/anon/pen/vgMaYX

If I can get to it tonight, I'll do a pull request, but no promises.

**Comment by shieldsjared** [comment]
Confirmed this bug as well.

**Comment by NonameSLdev** [comment]
Confirmed. Perhaps the names of the placeholders need a rename as it's unclear what each one does.

**Comment by acburst** [comment]
Fixed in 2acea468624f55370a1a15eb27c580d9db83454b
